addons/html_builder/static/src/plugins/website_info_option.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
move this into `html_builder/static/src/website_builder/`. this code should be part of the website addon, and we will move it there someday